tsconfig.json

```diff
@@ -44,8 +44,8 @@
     // "noResolve": true,                                /* Disallow 'import's, 'require's or '<reference>'s from expanding the number of files TypeScript should add to a project. */
 
     /* JavaScript Support */
-    // "allowJs": true,                                  /* Allow JavaScript files to be a part of your program. Use the 'checkJS' option to get errors from these files. */
-    // "checkJs": true,                                  /* Enable error reporting in type-checked JavaScript files. */
+     "allowJs": true,                                  /* Allow JavaScript files to be a part of your program. Use the 'checkJS' option to get errors from these files. */
+     "checkJs": true,                                  /* Enable error reporting in type-checked JavaScript files. */
```

Comment:
What's this for?

Reply:
In the `esbuild` that I copied from `pay.ubq` there is an import of a `js` file
```TypeScript
import extraRpcs from "../lib/chainlist/constants/extraRpcs";
```
doesn't work without these